src/main.rs

```diff
@@ -78,6 +78,12 @@ struct Cli {
     /// Ignore settings from .git-nbconfig.yaml
     ignore_git_nb_config: bool,
 
+    #[clap(long, action)]
+    /// Specify a regex to use to strip matching output. Even if other settings would keep
```

Comment:
nitpick: maybe note it's the rust dialect? Not obvious from how people install/use this